packages/core/src/Stores/client-store.js

```diff
@@ -2109,6 +2118,7 @@ export default class ClientStore extends BaseStore {
         localStorage.removeItem('isNewAccount');
         LocalStore.set('marked_notifications', JSON.stringify([]));
         localStorage.setItem('active_loginid', this.loginid);
+        localStorage.setItem('active_user_id', this.user_id);
```

Comment:
setting user_id into localStorage same like loginid in order to avoid losing its value on each page reload.